datalad/interface/results.py

```diff
@@ -48,9 +58,19 @@
 }
 
 
-def get_status_dict(action=None, ds=None, path=None, type=None, logger=None,
-                    refds=None, status=None, message=None, exception=None,
-                    error_message=None, **kwargs):
+def get_status_dict(
+    action: Optional[str] = None,
+    ds: Optional[Dataset] = None,
+    path: Optional[str] = None,
+    type: Optional[str] = None,
+    logger: Optional[logging.Logger] = None,
+    refds: Optional[str] = None,
+    status: Optional[str] = None,
+    message: str | tuple | None = None,
```

Comment:
is there a way to become more specific about types in tuple here (and `error_message` below), that it should have first element to be a `str`?

Reply:
Not that I'm aware of.

Reply:
I also didn't find :-/